patroni/config.py

```diff
@@ -202,12 +250,27 @@ class Config(object):
 
     def __init__(self, configfile: str,
                  validator: Optional[Callable[[Dict[str, Any]], List[str]]] = default_validator) -> None:
+        """Create a new instance of :class:`Config` and validate the loaded configuration using *validator*.
+
+        .. note::
+            Patroni reads the configuration from the command-line argument if it exists (given by *configfile*),
+            otherwise from the environment variable (see :cvar:`PATRONI_CONFIG_VARIABLE`).
+
+            Besides that Patroni also reads configuration values from all supported environment variables -- see
```

Comment:
We need somehow emphasize that environment variables are used only when there is no `PATRONI_CONFIGURATION` environment variable or when it doesn't contain a yaml document.

Reply:
Made a commit for this